src/app/shared/issue-tables/issue-tables.component.html

```diff
@@ -145,6 +145,19 @@
     </mat-cell>
   </ng-container>
 
+  <!--Tester Disagree Column-->
+  <ng-container matColumnDef="testerDisagree">
+    <th mat-header-cell *matHeaderCellDef mat-sort-header>Response</th>
+    <td mat-cell *matCellDef="let issue">
+      <span (click)="$event.stopPropagation()" *ngIf="issue.testerDisagree" [ngStyle]="this.labelService.setLabelStyle('ff9900')"
```

Comment:
Wondering if we can put the colour hex values in `label.service.ts`. Currently the colour values in `label.service.ts` are all in hex, which makes it a bit difficult to figure out what colours they are. Might be more helpful to define a bunch of constants e.g. 

```typescript
export const ORANGE: string = 'ff9900';
export const GREEN: string = '00802b';
```
then there's only one place to change if we want to change the label colours for some reason.

Reply:
Changes made!